scripts/qemu_launcher

```diff
@@ -45,7 +45,11 @@ usage() {
     echo -e "                          \tDefaults to 3333"
     echo -e "  -d | --dev              \tRun QEMU development image instead of normal image"
     echo -e "                          \tDefaults to core-image-pelux-minimal-dev-qemux86-64.ext4"
-    echo -e "  -s | --static           \tRun QEMU development image with static IP (192.168.7.2)"
+    echo -e "  -s | --static           \tThe switch to toggle running QEMU dev image with static IP (Defaults to false)"
+    echo -e "  -I | --target_ip        \tSpecify the target static IP (Defaults to 192.168.7.2)"
+    echo -e "                          \tPlease aware that to change the default static IP, it is also needed to update the"
+    echo -e "                          \tSTATIC_IP_ADDRESS in conf/variant/common/local.conf in meta layer and rebuild the image"
```

Comment:
I think it's better to just say update `local.conf` without stating location of that file (it's always in <builddir>/conf/ anyway)